### PR TITLE
Added promo video with banner img as thumbnail

### LINKED ISSUE #61: Incorporate promo video into site
#### Current Behavior
A new promotional video has been created for Meshery Playground - https://youtu.be/Do7htKrRzDA

#### Desired Behavior
Prominently present this video to site visitors.

#### Implementation
1. Allow Have this video play onclick of the site's large screenshot.
1. Add a "play" icon on top of the site's screenshot, so that users understand that a video will load on click.
1. The video should be embedded either directly in the page or be presented in a modal.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description:**
Added the promo video with the current screenshot as the thumbnail using the `react-player` package.
Also, retained the styles to the banner :)

This PR fixes #61 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
No you didn't...  Ha! Yes, you did. Oh, that is very cool how you retained the banner styles behind the video. 😁

File: site/src/App.js [hunk 1]
```diff
@@ -49,11 +50,17 @@ const App = () => {
...
+              url="https://www.youtube.com/embed/Do7htKrRzDA"
+              playing
+              controls
+              light={mesheryPlayground}
+              width="90%"
+              style={{margin: "auto"}}
+              className="embedVideo"
+            />
+            <p className="caption">Launch and learn with Meshery</p>
+            <p className="byline">The cloud native playground contains a set of learning paths that incorporate an application networking centric curriculum featuring training on 100+ integrations.</p>
```
Comment (leecalcote) [inline site/src/App.js:63]:
```suggestion
            <p className="byline">The cloud native playground contains a set of learning paths that incorporate an application networking centric curriculum featuring training on 150+ integrations.</p>
```

**Comment by leecalcote** [comment]
I wonder if there's a way that we might make the _play_ triangle catch the visitor's eye a little more than is does now.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Or maybe if there were a bit of an onHover effect (outside of the pointer) on the banner.

**Comment by Nikhil-Ladha** [comment]
> Or maybe if there were a bit of an onHover effect (outside of the pointer) on the banner.

Probably, we could do both. Let me see...

**Comment by Nikhil-Ladha** [comment]
How about now? 👀

**Comment by Nikhil-Ladha** [comment]
Merging it, considering the priority of it.
If any change is needed, let me know :)

**Comment by leecalcote** [comment]
Thank you, thank you. 👏

**Pull request merged**